generate_wordstr_box.py

```diff
@@ -32,11 +32,13 @@
 # load gt
 with io.open(args.txt, "r", encoding='utf-8') as f:
     lines = f.read().strip().split('\n')
+    if len(lines) > 1:
+        raise ValueError("ERROR: %s: Ground truth text file should contain exactly one line, not %s" % (args.txt, len(lines)))
+
 
 # create WordStr line boxes for Indic & RTL
-for line in lines:
-    line = unicodedata.normalize('NFC', line.strip())
-    if line:
-        line = bidi.algorithm.get_display(line)
-        print("WordStr 0 0 %d %d 0 #%s" % (width, height, line))
-        print("\t 0 0 %d %d 0" % (width, height))
+if line:
+    line = unicodedata.normalize('NFC', lines[0].strip())
+    line = bidi.algorithm.get_display(line)
```

Comment:
Moving the conditional `if line:` changes the behaviour. Lines with only whitespace are now handled. I don't think that's what we want here.

Reply:
oh, right, I resolved that incorrectly. will fix.

Reply:
271d6fa